src/hb-ot-cmap-table.hh

```diff
@@ -272,17 +272,12 @@ struct CmapSubtableLongSegmented
   }
 
   inline bool serialize(hb_serialize_context_t *context,
-                        unsigned int group_count,
-                        Supplier<CmapSubtableLongGroup> &group_supplier)
+                        hb_prealloced_array_t<CmapSubtableLongGroup> &group_data)
   {
     TRACE_SERIALIZE (this);
     if (unlikely(!context->extend_min (*this))) return_trace (false);
-    if (unlikely(!groups.serialize(context, group_count))) return_trace (false);
-    for (unsigned int i = 0; i < group_count; i++)
-    {
-      const CmapSubtableLongGroup &group = group_supplier[i];
-      memcpy(&groups[i], &group, sizeof(group));
-    }
+    Supplier<CmapSubtableLongGroup> supplier(group_data.array, group_data.len);
+    if (unlikely(!groups.serialize(context, supplier, group_data.len))) return_trace (false);
     return true;
```

Comment:
Missing space before `(` (and in lots more places).

Reply:
Fixed some. Doubtless not all :D